### PR TITLE
Fix new GitHub issues

### LINKED ISSUE #1: Feat: Change the Issue/PR template to the newer form versions
## Description

GitHub has introduced new issue template features. Now Issues templates can be forms, we can Implement that in this project as well what do you think?

### PR DESCRIPTION
**Pull request by krishdevdb**
## Description

- [x] I have followed the [code of conduct](https://github.com/krishdevdb/discord-plutus/blob/main/contributing.md)
- [x] I have followed the [contributing guidelines](https://github.com/krishdevdb/discord-plutus/blob/main/contributing.md).
- [x] Either I have tested the code or my changes are not related to code.

## Related Issue

fixes #1 

## Context

> GitHub has introduced new issue template features. Now Issues templates can be forms, we can Implement that in this project as well what do you think?

## How Has This Been Tested?

This meets the GitHub Documentation Specs

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Documentation
+description: File a documentation request/report
+title: "[Docs:]: "
```
Comment (krishdevdb) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: Maybe I misspelled these
```suggestion
title: "[Feat]: "
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Documentation
```
Comment (krishdevdb) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]: Looks like I messed up with the file names

**Comment by kaiwalyakoparkar** [comment]
LGTM :rocket:

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Pull request merged**